xchange-hitbtc/src/main/java/org/knowm/xchange/hitbtc/v2/internal/HitbtcAdapters.java

```diff
@@ -241,21 +242,70 @@ public static ExchangeMetaData adaptToExchangeMetaData(List<HitbtcSymbol> symbol
 
   public static FundingRecord adapt(HitbtcTransaction transaction) {
 
-    //todo: find out if there are more statuses
-    FundingRecord.Status status = transaction.getStatus().equals("success") ?
-        FundingRecord.Status.COMPLETE :
-        FundingRecord.Status.FAILED;
-
+    String description = transaction.getType() + " " + transaction.getStatus();
+    if (transaction.getIndex() != null) {
+        description += ", index: " + transaction.getIndex();
+    }
+    if (transaction.getHash() != null) {
+        description += ", hash: " + transaction.getHash();
+    }
+    if (transaction.getPaymentId() != null) {
+        description += ", paymentId: " + transaction.getPaymentId();
+    }
+    
```

Comment:
Can we use `transaction.getHash()` for the externalId in fundingRecord?  I think that's what a few other implementations have done where it's available?

Reply:
sure, we can.
there is nothing to be said against it.